apps/src/codebridge/InfoPanel/MainInstructionsBubblePreview.tsx

```diff
@@ -0,0 +1,42 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import {Theme} from '@cdo/apps/lab2/types';
+
+import MainInstructionsContent from './MainInstructionsContent';
+
+import moduleStyles from '@codebridge/InfoPanel/styles/validated-instructions.module.scss';
+
+interface MainInstructionsBubblePreviewProps {
+  instructionsText: string;
+  handleInstructionsTextClick?: (id: string) => void;
+  theme: Theme;
+  hasPassed: boolean;
+}
+
+/**
+ * Component for levelbuilder to preview the main instructions bubble (without predict answers or validation).
+ */
+const MainInstructionsBubblePreview: React.FunctionComponent<
```

Comment:
Curious about the name 'MainInstructionsBubblePreview'...why did you include 'Bubble' in name?

Reply:
We styled the instructions with the name "bubble" in the css styles, because we have multiple sections of the instructions (ex. main text, validation results, continue button)

Reply:
discussed offline, renamed!